### ISSUE TITLE
changing club

### ISSUE THREAD

**Issue by JennMart** [sharkbit/BadgeApp#148]
I renewed someone this morning and he added a second club and I confirmed both his clubs were current.  I saved/renewed his badge and when I looked at the range badge list he was listed with a different club (APL).  I looked over the range badge list and found there are several recent badge renewals listed as APL membership.  One was a name I recognized and remembered the club so I corrected it but it looks like somehow other club entries are being saved as APL.  They typically have very few members but are suddenly listed with a bunch.

**Comment by JennMart** [comment]
This looks like it started on the 21st.  It just happened again with a renewal we just did.  When you update it saves correctly and I checked the new badges that were sold this weekend and they are correct.  It looks like it is changing every renewal to APL.  I updated those that I was able to determine the clubs but there are several we don't have correct clubs for.   I can call them individually to get their clubs corrected this weekend if I need to.
